hl7-mllp/2.1/modules/ROOT/edits/hl7-mllp-connector-examples.adoc

```diff
@@ -0,0 +1,136 @@
+= HL7 EDI - Examples - Mule 4
+
+Use these examples to read and write HL7 messages over the MLLP protocol.
+
+== Example: HL7 Studio
+
+Set these Global properties before using the examples:
+
```

Comment:
Reformat this list to our standard for bulleted lists, per style reference.

Reply:
fixed.